kinto/__main__.py

```diff
@@ -4,73 +4,93 @@
 import sys
 import logging
 import logging.config
-from six.moves import input, configparser
+
 from kinto.core import scripts
 from pyramid.scripts import pserve
 from pyramid.paster import bootstrap
 from kinto import __version__
 from kinto.config import init
 
-CONFIG_FILE = 'config/kinto.ini'
+DEFAULT_CONFIG_FILE = 'config/kinto.ini'
+DEFAULT_PORT = 8888
+DEFAULT_LOG_LEVEL = logging.INFO
+DEFAULT_LOG_FORMAT = "%(levelname)-5.5s  %(message)s"
```

Comment:
Usually jobs runner takes care of this. This command is meant to be run by hand and it wanted to have the most usuful output.